docs/_layouts/datasource.html

```diff
@@ -49,9 +46,11 @@
 <!--.sitewide includes-->
 {% include footer-includes.html %}
 {{page.javascript}}
+<!-- KM question: what does {{page.javascript}} do? -->
```

Comment:
@marwahaha anywhere you see a `{{page.whatever}}` it's where it inserts the variable from the frontmatter. This was intended to be somewhere that you could put page-specific script tags so they would be at the bottom of the page. This would just be if you needed something that applied to one page, not the whole layout i.e. the crime page gets some extra JS - since all our datasource files need your datasource.js script that's in the right spot.